src/components/StatePicker.js

```diff
@@ -15,30 +16,57 @@ const propTypes = {
     label: PropTypes.string,
 
     /** A callback method that is called when the value changes and it received the selected value as an argument */
-    onChange: PropTypes.func.isRequired,
+    onInputChange: PropTypes.func.isRequired,
 
     /** The value that needs to be selected */
     value: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
 
+    /** Indicates that the input is being used with the Form component */
+    isFormInput: PropTypes.bool,
+
+    /**
+    * The ID used to uniquely identify the input
+    *
+    * @param {Object} props - props passed to the input
+    * @returns {Object} - returns an Error object if isFormInput is supplied but inputID is falsey or not a string
+    */
+    inputID: props => FormUtils.validateInputIDProps(props),
+    
+    /** Saves a draft of the input value when used in a form */
+    shouldSaveDraft: PropTypes.bool,
+
+    /** Callback that is called when the text input is blurred */
+    onBlur: PropTypes.func,
+
+    /** Error text to display */
+    errorText: PropTypes.string,
+   
     ...withLocalizePropTypes,
 };
 
 const defaultProps = {
     label: '',
     value: '',
```

Comment:
Maybe pass `undefined`. `''` is still a value. I am not sure if Picker checks for truthy value before binding it.

Reply:
I am talking about the value.

Reply:
yes value will be undefined here

Reply:
How? you are passing `''`